src/engine/database/mod.rs

```diff
@@ -52,6 +52,24 @@ pub enum DatabasePool {
     NoDatabase,
 }
 
+impl DatabasePool {
+    #[cfg(feature = "neo4j")]
+    pub(crate) fn neo4j(&self) -> Result<&bb8::Pool<bb8_bolt::BoltConnectionManager>, Error> {
```

Comment:
I changed these because I realize this isn't really an `into` kind of function.  It's not a type conversion... just getting a connection from a pool.  But, I already commented asking for one change here, and I didn't want to do a "no, the couch really looked better where it was," routine, so I fixed and pushed this one.